### ISSUE TITLE
Change name of App to something "Contact" related

### ISSUE THREAD

**Issue by bnvk** [open-keychain/open-keychain#1208]
I proposed this idea to @dschuermann and @Valodim at the OpenPGP Summit and both seemed warm and open to the idea.

A discussion in the [@modernpgp/terminology](https://github.com/ModernPGP/terminology) working group, there is the idea to change the word **"keychain"** to **"contacts"** as it is more human friendly and understandable. Here are a few ideas I have (and reasoning behind them).

* Key Contacts (sounds too simple)
* PGP Contacts (very specific)
* Open Contacts (does not hint at security)
* Crypto Contacts (specific + lends to supporting other crypto)
* Secure Contacts (has a descriptive but user friendly feel)

I realize this is a pretty major change for a project of your adoption + maturity to undergo, but I just wanted to file this issue to the keep the discussion / idea going :smile:

**Comment by dschuermann** [comment]
After getting some sleep thinking about this, I tend to go on gradually with this.
Maybe we can switch to "contacts" in the UI in a first step and see how this works out in practice.

**Comment by bnvk** [comment]
@dschuermann even that would be a great step forward IMHO, and definitely probably less jarring to users. Good call :+1:

**Comment by Valodim** [comment]
If the primary unit we have are contacts rather than keys, shouldn't a contact be able to have more than one key? Not saying we should do that (the complexity!), just throwing it out there that this might be an expectation...

**Comment by bnvk** [comment]
@Valodim yes, that is exactly what I think we should do- shift the perception away from "Keys" and move towards "Contacts" whereby a "Contact" in my address book has multiple keys which i can choose to add / remove / mark as do not use / encrypt to / etc...

Hehe, definitely some complexity there especially with the UID's which are supposed to present multiple identities... fun times ahead :grinning:

**Comment by Valodim** [comment]
If we decide that a 1-n contact-key structure is what we want, it makes sense to tie this major change in semantics to the change of name. Looking at our current schedule of releases, could be a long time til we get there
